packages/next/compiled/terser-webpack-plugin/minify.js

```diff
@@ -2,7 +2,7 @@
 
 const {
   minify: terserMinify
-} = require("next/dist/compiled/terser");
+} = require('terser');
```

Comment:
Why was this undone?

Reply:
Nicely spotted! I've added this back. It appears this is an asset emission of the ncc build which therefore does not follow the externals definition.

The original change must have been a manual fix. We should automate this into the compilation somehow ideally....

Reply:
Posted https://github.com/vercel/ncc/issues/611.